Cilsil/Cil/Parsers/ExceptionParser.cs

```diff
@@ -0,0 +1,293 @@
+using Cilsil.Utils;
+using Cilsil.Sil;
+using Cilsil.Sil.Types;
+using Mono.Cecil.Cil;
+using Mono.Cecil;
+using Cilsil.Sil.Instructions;
+using Cilsil.Sil.Expressions;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Cilsil.Cil.Parsers
+{
+    internal class ExceptionParser : InstructionParser
+    {
+        protected override bool ParseCilInstructionInternal(Instruction instruction,
+                                                            ProgramState state)
+        {
+            var endBlockOffset = state.ExceptionBlockStartToEndOffsets[state.CurrentInstruction.Offset];
+            
+            /*node1 preds: succs:2 3 exn: Instructions
+            n$47=*&amp;return:void [line 55];   # load return
+            *&amp;return:void=null [line 55];
+            n$48=_fun___unwrap_exception(n$47:void) [line 55];*/
+
+            // Create exception handler node.
```

Comment:
comment isn't very useful as I can gather as much from the variable name/code line, i'd generally remove comments which don't add too much value

Reply:
That's not to say I think that you need fewer comments, but they should be more descriptive. For example, tell me what a catch variable semantically means/how it's used/what is significant about it within the pipeline.

Reply:
Done